android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
> there's some sort of limit on how many bytes you can allocate on a RN string

:/

That's quite something to deal with.

Well since the issues could block other people i guess returning path to **RN** will help and it don't costs too mutch.